app/javascript/src/Policies/reducers/PolicyChain.jsx

```diff
@@ -22,6 +22,12 @@ function addPolicy (state: ChainState, action: AddPolicyToChainAction): ChainSta
 }
 
 function updateChain (_state: ChainState, action: UpdatePolicyChainAction): ChainState {
+  // HACK: enable the submit button after any change is made
```

Comment:
Hmmm, if this is a reducer then it shouldn't modify the dom.
If this hack is urgent, then please create a task to improve it

Reply:
I think it's going to be removed whenever we refactor the policies anyways. The reason to do it this way it is not to add much more complexity for a thing that.. shouldn't be done, at all.. for starters the submit button should be part of the React component.

Reply:
Yes, but is not clear what `whenever we refactor` means, it may be next week or next year. If we have a task to refactor policies, just add a comment, if not create a new one

Reply:
This is way to hacky. Ideally we could check the pristine state with Redux, but if we can't...what about saving the original json (the proxy config generated) and compare it when rendering the form? That'd be only a string comparison, not so expensive

Reply:
I can try that